public/apps/account/account-nav-button.tsx

```diff
@@ -128,7 +128,7 @@ export function AccountNavButton(props: {
       >
         View roles and identities
       </EuiButtonEmpty>
-      {isMultiTenancyEnabled && (
+      {isMultiTenancyEnabled && props.config.multitenancy.enabled && (
```

Comment:
Sure will do. Thanks.

Reply:
@DarshitChanpura `isMultiTenancyEnabled ` can change if we change the value of multitenancy_enabled in Security config. Therefore making a constant using it might not be the best idea.